### DISCUSSION TITLE
Disable mouse scroll input

### DISCUSSION THREAD

**Discussion by wgmayer0** [qutebrowser/qutebrowser#7465] [General]
I use [keynav](https://github.com/jordansissel/keynav) to browse the web with qutebrowser. I made the `<Ctrl-j>` and `<Ctrl-k>` keys bind to `xdotool click 5` and `xdotool click 4` respectively in my keynavrc (no bindings in qutebrowser). When I use those key combinations and send input to qutebrowser, qutebrowser zooms in/out. How do I disable the zoom functionality from the scroll so that qutebrowser just receives a scroll?

**Comment by toofar** [comment]
I guess it's [picking up](https://github.com/qutebrowser/qutebrowser/blob/7f693f7/qutebrowser/browser/eventfilter.py#L153) the Ctrl that is still pressed. I'm not sure how you could get around that. I don't support telling xdotool to send some keyboard keys that are mapped to a scroll command in qutebrowser would work?

**Comment by wgmayer0** [comment]
That's a good idea I will try it. The question is, what is the best option?
I'll read thru the docs and try to find something. The only other problem
is outside of qutebrowser I wont be able to scroll using keynav

**Reply by toofar** [reply]
Probably just the Up and Down arrow keys? Or if you want to scroll in insert mode you could bind something to `scroll up|down` in insert mode. Probably will have to make the bindings Ctrl+whatever though.
